tests/bootstrap.php

```diff
@@ -9,6 +9,6 @@
  * file that was distributed with this source code.
  */
 
-$loader = require_once __DIR__ . "/../vendor/autoload.php";
+$loader = require __DIR__ . "/../vendor/autoload.php";
```

Comment:
The test failed when set to require once. I can't remember the exact error or the explanation for it, but this fixed it. Plus it's only in the tests that this has been changed, so not too much to worry about... I can try and reproduce the error if you want, but I am certain that it wasn't something that I later discovered that was wrong with my environment...